app/scripts.babel/popup.js

```diff
@@ -194,6 +193,10 @@ limitations under the License.
       if (app.browser.name === 'Firefox') {
         adjustStorageButtons();
       }
+      // BDE-187: Hide the Designer Live Preview button if the current browser is Chrome
+      if (app.browser.name === 'Chrome') {
```

Comment:
would be more interesting to use `!== firefox`, no? To cover `Chromium`, and consor?